app/app.go

```diff
@@ -737,6 +712,8 @@ func New(
 		authzmodule.NewAppModule(appCodec, app.AuthzKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
 	)
 
+	app.ModuleBasics = ModuleBasics
```

Comment:
Why do we need this change by setting ModuleBasics in app?

Reply:
The default genesis function is now part of module basics , which we use in the simulation tests 
// DefaultGenesis provides default genesis information for all modules
```
func (bm BasicManager) DefaultGenesis(cdc codec.JSONCodec) map[string]json.RawMessage {
	genesis := make(map[string]json.RawMessage)
	for _, b := range bm {
		if mod, ok := b.(HasGenesisBasics); ok {
			genesis[b.Name()] = mod.DefaultGenesis(cdc)
		}
	}

	return genesis
}
```

Reply:
Do we still need `ModuleBasics` as a public variable?

Reply:
you are correct, we dont 
f8c095d8b9820bc8de07ac0e29c30dce0f991d8c